components/script_layout_interface/reporter.rs

```diff
@@ -23,11 +23,11 @@ pub struct CSSErrorReporter {
 
 impl ParseErrorReporter for CSSErrorReporter {
      fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str,
-         servo_url: &ServoUrl) {
+         _url: &ServoUrl) {
```

Comment:
Thank you! Although in the cases where url is used in the function, it shouldn't be prefixed with an underscore. The underscore tells the compiler that the variable is not intended to be used in the function so the compiler knows not to emit an unused variable warning